website/server/models/user/schema.js

```diff
@@ -521,6 +521,7 @@ let schema = new Schema({
     return {};
   }},
   pushDevices: [PushDeviceSchema],
+  ABtest: {type: String},
```

Comment:
Are we only ever going to have a single AB test going on? Should we at least do:

```js
ABtest: {
  newUserRetention: String
}
```

As I assume there are other kinds of AB testing we could do.